src/discretization.jl

```diff
@@ -1,8 +1,6 @@
 
-struct Discretization{LType, UType, PType}
-  low_mat::LType                        # Lower triangular matrix of LU composition
-  upp_mat::UType                        # Upper triangular matrix of LU composition
-  perm_array::PType                     # Permutation array of LU composition
+struct Discretization{DecType}
+  lu_dec::DecType                       # LU decomposition of system matrix
```

Comment:
```suggestion
struct Discretization{DecompositionType}
  lu_decomposition::DecompositionType                       # LU decomposition of system matrix
```
I am always in favor of descriptive, non-abbreviated names for better understandability 😬

Reply:
Thanks!